src/azure-cli-core/setup.py

```diff
@@ -57,16 +57,13 @@
     'packaging>=20.9,<22.0',
     'paramiko>=2.0.8,<3.0.0',
     'pkginfo>=1.5.0.1',
+    # psutil can't install on cygwin: https://github.com/Azure/azure-cli/issues/9399
+    'psutil~=5.9; sys_platform != "cygwin"',
     'PyJWT>=2.1.0',
     'pyopenssl>=17.1.0',  # https://github.com/pyca/pyopenssl/pull/612
     'requests[socks]'
 ]
 
-# dependencies for specific OSes
-if not sys.platform.startswith('cygwin'):
-    DEPENDENCIES.append('psutil~=5.9')
```

Comment:
These lines are added by https://github.com/Azure/azure-cli/pull/15076.

This brought back https://github.com/Azure/azure-cli/issues/9399, making it impossible to install `azure-cli` on cygwin:

```
Collecting psutil~=5.9
  Using cached psutil-5.9.0.tar.gz (478 kB)
  Preparing metadata (setup.py) ... error
  error: subprocess-exited-with-error

  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> [1 lines of output]
      platform cygwin is not supported
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
```

Reply:
This is what it becomes in generated wheel `MEDADATA`:

```
Requires-Dist: psutil (~=5.9) ; sys_platform != "cygwin"
```

Reply:
Well, anyway, latest `cryptography` doesn't have cygwin support either: https://github.com/pyca/cryptography/issues/6834